readingDataPointsFilters:
  - RandomSamplingDataPointsFilter:
      prob: 1.0

referenceDataPointsFilters:
#  - MaxDistDataPointsFilter:
#      maxDist: 20.0
  - RandomSamplingDataPointsFilter:
      prob: 1.0

matcher:
  KDTreeMatcher:
    knn: 5
    maxDist: 1.0
    epsilon: 0

outlierFilters:
  - TrimmedDistOutlierFilter:
     ratio: 0.58
  - SurfaceNormalOutlierFilter:
     maxAngle: 1.57
 # - GenericDescriptorOutlierFilter:
 #    source: reference
 #    descName: probabilityStatic

errorMinimizer:
  PointToPlaneErrorMinimizer

transformationCheckers:
  - DifferentialTransformationChecker:
      minDiffRotErr: 0.01
      minDiffTransErr: 0.02
      smoothLength: 2
  - CounterTransformationChecker:
      maxIterationCount: 20
  - BoundTransformationChecker:
      maxRotationNorm: 0.8
      maxTranslationNorm: 20.00

inspector:
#	VTKFileInspector
	NullInspector

logger:
#	FileLogger
	NullLogger
